src/types/product.d.ts

```diff
@@ -0,0 +1,8 @@
+export type Product = {
+  id: String;
+  name: String;
+  description: String;
+  price: Number;
+  stock: Number;
+  category: String;
```

Comment:
this is out of date, sorry, should have kept you in the loop about category/categories 😬

Reply:
Fixed it!